examples/notebooks/README.md

```diff
@@ -20,3 +26,11 @@ pip install notebook scikit-learn-intelex
 | KNeighborsClassifier   |       MNIST     | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/knn_mnist.ipynb) |    [sklearn.neighbors.KNeighborsClassifier](https://scikit-learn.org/stable/modules/generated/sklearn.neighbors.KNeighborsClassifier.html) |
 |        NuSVR           | Medical charges | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/nusvr_medical_charges.ipynb) | [sklearn.svm.NuSVR](https://scikit-learn.org/stable/modules/generated/sklearn.svm.NuSVR.html) |
 | RandomForestRegressor  |     Yolanda     | [View source on GitHub](https://github.com/intel/scikit-learn-intelex/blob/master/examples/notebooks/random_forest_yolanda.ipynb) | [sklearn.ensemble.RandomForestRegressor](https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.RandomForestRegressor.html) |
+
+<details><summary>[Click to expand] ℹ️ Time measurements in python notebooks </summary>
+  
+Training and inference times are measured using the [**%%time**](https://ipython.readthedocs.io/en/stable/interactive/magics.html#magic-time) function. This shows 2 times:
+- CPU times: **sys** - the operating system CPU time due to system calls from the process;  
```

Comment:
```suggestion
- CPU times: **sys** - the operating system CPU time due to the system calls from the process;  
```

Reply:
Also, maybe divide it like that: 
-CPU times:
  - **sys**-......
  - **user**-....

-**Wall time** - ....
It will not confuse users, since we mention that it appears 2 times, but here are 3 bullets.